custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc.py

```diff
@@ -245,18 +245,19 @@ def updates(self):
         yield """
         CREATE UNLOGGED TABLE "tmp_household" AS SELECT
             owner_id,
-            sum(open_count) AS cases_household
+            sum(open_count) AS cases_household,
+            count(*) AS all_cases_household
         FROM "{household_cases}"
         WHERE opened_on<= %(end_date)s
         GROUP BY owner_id;
         UPDATE "{tablename}" agg_awc SET
            cases_household = ut.cases_household,
-           is_launched = CASE WHEN ut.cases_household>0 THEN 'yes' ELSE 'no' END,
```

Comment:
There are lots of test cases getting failed if I m adding a test to check this. So putting out the code here so that @calellowitz / @czue  you can review and we can unblock the change.

I am meanwhile trying to fix the test cases.